.github/pull_request_template.md

```diff
@@ -1,24 +1,33 @@
-# Description
+## Description
 
-- Provide a summary of the modification as well as the issue that has been resolved. List any dependencies that this modification necessitates.
+Provide a summary of the modification as well as the issue that has been resolved.
 
 - Fixes # (issue)
 
 ## Changes
 
+<details>
+<summary>Describe the changes you made</summary>
+
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which refactors the code base)
+- [ ] Enhancement (non-breaking change which improves the code base)
```

Comment:
How is enhancement different from refactor or new feature? In my head enhancement = new feature + refactor

Reply:
that's a fair point yeah